service/block/types.go

```diff
@@ -10,23 +11,33 @@ import (
 // it is still awaiting erasure coding.
 type Raw = core.Block
 
-// ExtendedBlock contains the erasure coded block data as well as its
+// Block represents the entirety of a Block in the Celestia network.
+// It contains the erasure coded block data as well as its
 // ExtendedHeader.
-// TODO @renaynay: ExtendedBlock still needs to be spec'd out.
-type ExtendedBlock struct {
-	// TODO @renaynay: include pointer to ExtendedHeader
-	data *ExtendedBlockData
+type Block struct {
+	header     *header.ExtendedHeader
+	data       *ExtendedBlockData
+	lastCommit *core.Commit
 }
 
 // ExtendedBlockData is an alias to rsmt2d's ExtendedDataSquare type.
-// TODO @renaynay: ExtendedBlockData still needs to be spec'd out.
 type ExtendedBlockData = rsmt2d.ExtendedDataSquare
 
-func (e *ExtendedBlock) Data() *ExtendedBlockData {
-	return e.data
+// Header returns the ExtendedHeader of the Block.
+func (b *Block) Header() *header.ExtendedHeader {
+	return b.header
+}
+
+// Data returns the erasure coded data of the Block.
+func (b *Block) Data() *ExtendedBlockData {
+	return b.data
+}
+
+// LastCommit returns the last commit of the Block.
+func (b *Block) LastCommit() *core.Commit {
```

Comment:
Alias for commit?

Reply:
Are we gonna use it that much throughout the codebase that it needs an alias?

Reply:
Not sure. Let's introduce one once we use it somewhere else.

Reply:
Resolving for now